### PR TITLE
New pages, nav updates, bug fixes

### PR DESCRIPTION
**Pull request by JoshMountain**

### CODE REVIEW

**Comment by brikis98** [comment]
Took a quick look and some thoughts/feedback, primarily for @josh-padnick: 

1. Footer still has old items. What do we want in the footer?
1. It looks like this is still the old How it Works page. Weren't we going to redesign this? Or are we leaving it as-is and leaving the redesign for later?
1. What is the Usage Patterns Library page going to have? I know we'll be moving to self-service later, but what are we offering for that *now*?
1. Are the training video course stats correct (e.g., 40 videos, 122 minutes)?
1. I'm bothered that none of the images/headers on the training page are clickable. I guess we don't want people to navigate away from the page, but it's really weird.
1. What will be on the DevOps Bootcamp page?
1. What will be on the Custom Module development page?

**Comment by josh-padnick** [comment]
> Footer still has old items. What do we want in the footer?

Here's my take. Feel free to modify:

- Products & Services
  - Infrastructure Code Library
  - Usage Patterns Library
  - DevOps Training Library
  - Support
  - DevOps Bootcamp
  - Custom Module Development

- Learn
  - How It Works
  - Pricing
  - FAQ
  - Customers
  - Why Prod is Down

- Company
  - About
  - Blog
  - Careers

- Connect
  - Contact Us
  - Newsletter
  - Community Forum
  - Social Links

> It looks like this is still the old How it Works page. Weren't we going to redesign this? Or are we leaving it as-is and leaving the redesign for later?

The thought here was that to save time and get this launched, we should just use the old how it works design, but add new content. Unfortunately, it looks like the new How It Works page didn't make it into the build out. 

@JoshMountain Can you check w/ Andrew and do the build out for the new How It Works content and icons please?

> What is the Usage Patterns Library page going to have? I know we'll be moving to self-service later, but what are we offering for that now?

The thought was that we'd expose our infra-live and infra-modules repos that include just about every option we offer in the Reference Architecture. This makes the whole offering more valuable. For now, I'd just write some copy that describes this. Might be nice to include some kind of in-style file layout that shows the pattern as an image, but that's only if you have the time.

> Are the training video course stats correct (e.g., 40 videos, 122 minutes)?

I assume not. I think we'll have to manually compile these.

>  I'm bothered that none of the images/headers on the training page are clickable. I guess we don't want people to navigate away from the page, but it's really weird.

Now that I see this in action, I agree, it is awkward. We do have a Preview link that's meant to go to Teachable so users can directly watch some preview videos. See my comments below on some additional details on this. Perhaps we should add some kind of hover effect on the image so that it says "Watch a Preview" when you hover over it and it takes you to Teachable.

> What will be on the DevOps Bootcamp page?

We should just repurpose the content from our [current training page](https://gruntwork.io/training/). The idea is that if someone wants the onsite training, this is where we give them all the details around that. 

The goal of these pages isn't to be a comprehensive summary of everything we include in the Bootcamp, just enough that we convey the core ideas so that if someone's interested in they can reach out to see if it's the right fit.

> What will be on the Custom Module development page?

Here we want to appeal to ISVs and possibly cloud vendors. We should talk about our business approach to building new Terraform modules, what criteria we use to accept projects, how we support it afterward, how we charge for it, etc. Include a call-to-action to contact us for more info.

> A few of the new pages have placeholder text that needs to be replaced: pricing page (most sections), training page (live training blurb).

Yeah, it seems the easiest way to do this sometimes is to just fill it in directly.

### Other Thoughts

#### Training Images are Low-Res

Images like this one appear to be pretty low-res:

image

Can you fix this so that these look crisp?

#### Teachable links appear to be off

@JoshMountain The links on the /training page should point directly to the Teachable preview ([example for Terraform](https://training.gruntwork.io/courses/terraform/lectures/3429337)). Also, my understanding was that you were going to modify Teachable so that its look and feel would be consistent with the site (to a minimal extent) and more importantly that the "back" links would go back to our main site. None of this appears to be implemented. Can you advise on next steps here?

#### We should add some new clients to the webpage

Some of the clients listed here have gone out of business or for whatever reason aren't with us anymore. Let's update to some of our newer clients.

#### Taking the lead on changes

I have a couple quick changes that OCD requires me to make which I'll do now. Otherwise, @brikis98, much appreciated for taking the lead on this.

**Comment by josh-padnick** [comment]
@JoshMountain One other issue I noticed is that the training library spacing doesn't match up with the design:

Design:
image

HTML:
image

Can you fix?

Finally, I did want to say thank you for making all these changes. The pricing page, in particular, looks great. In future Pull Requests, you should be sure to call out exactly what you added. I don't think it was clear to our team that as part of this PR you also fixed some mobile rendering issues and improved our contact us form handling.

**Comment by brikis98** [comment]
Note: I'm currently working my way through the various pages to fill in or update the content. I've done the IaC Library and Usage Patterns Library pages so far.

**Comment by brikis98** [comment]
OK, just filled in the content for the training page. I pulled out the resources into a separate DevOps Resources page, which I think is much cleaner.

**Comment by brikis98** [comment]
OK, most pages are now up to date.

Remaining work:

1. The How it Works page needs its new design. @JoshMountain can you update us on this?
1. The training videos need proper preview pages. @JoshMountain can you update us on this?
1. Final review of text/design. @josh-padnick, @mcalhoun I'd appreciate a second set of eyes on this.

I would like to get this pushed live this week, so everyone, please take a look as soon as you can. Thanks!

**Comment by JoshMountain** [comment]
@brikis98 I've added a few commits that should take care of 1 and 2 as well as a the blurry images and sizing on the Training page. Let me know if you have any questions!

**Comment by brikis98** [comment]
@JoshMountain Thanks, looks great!

@josh-padnick and @mcalhoun I've updated all the contents on the How it Works page now too. I think it's much clearer. I would like to push this website live TODAY. Please take a look and provide feedback!

**Comment by josh-padnick** [comment]
@JoshMountain Thanks for all the updates! It's a nice improvement.

On the Teachable piece, I think we do need to do that. Otherwise, it's very hard to get back to our site. Here are the specific updates that make sense to me:

1. The top-left link on the **unauthenticated** page should go back to the Training page on gruntwork.io. It'd be especially nice if we could somehow communicate to the user that goes back to gruntwork.io. Note that for **authenticated** pages, we should leave this alone:

image

2. When the user views a video that's not available in preview, the button they see should say: "Join Gruntwork to Unlock". The link should go to the "Contact Us" page.

image

1. If possible, it'd be nice if the colors matached our website, but like Jim said, that's not essential.

Thanks!

**Comment by brikis98** [comment]
@josh-padnick It would be nice if the page took you back to gruntwork.io, but it's not a blocker for launch.

**Comment by josh-padnick** [comment]
@brikis98 Ok, fair enough. Seemed important to me (our eternal debate), but I'm good to launch before this is done as long as we agree to do it soon.

**Comment by josh-padnick** [comment]
@JoshMountain Just one more set of update requests:

1. Can you make new customer logos for the following please?:
  - Urban Footprint (https://urbanfootprint.com/) Formerly "Calthorpe Analytics"
  - Communication Service for the Deaf (http://csd.org)
  - Finiata (https://www.finiata.com/)

Let's remove:
- Bolste
- Rhumbix
- Proxy Project

1. On the "How it Works" page, for the first icon to the left of "You Need to Run Your Infra on AWS", can we use a light bulb instead of a question mark here?

I've now completed a comprehensive review and those are all the remaining changes I've got. In line with the above discussion, these are timely but not urgent and we can launch without these.

**Comment by brikis98** [comment]
Alright, merging and deploying!

**Comment by brikis98** [comment]
@JoshMountain Once all of this work is done, I'd be grateful if you could remove the gruntwork-io website from the User10 repos so there are no SEO issues. Thx!

File: pages/initial-setup/index.html [hunk 1]
```diff
@@ -1,12 +1,11 @@
 ---
 layout: default
 title: Initial Setup &amp; Deployment
-excerpt: We write the Terraform code for you, apply it to your AWS accounts, and push it to your git repos.
+excerpt: We define your entire infrastructure as code and deploy it into your AWS accounts in about 1 day.
 permalink: /initial-setup/
 slug: subscriptions
 footer_heading: Talk with our team of DevOps experts now!
-redirect_from:
-  - /reference-architecture/
```
Comment (josh-padnick) [inline pages/initial-setup/index.html]: Isn't this the new RefArch page?

Comment (brikis98) [reply pages/initial-setup/index.html]: I made it the Usage Patterns Library page

**Pull request merged**